DER/ExchangeTradedDerivatives/ExchangeTradedOptions.rdf

```diff
@@ -707,5 +721,23 @@
 		<rdfs:range rdf:resource="&fibo-fnd-utl-alx;Percentage"/>
 		<skos:definition xml:lang="en">Predetermined spread at which the holder may exercise the contract.</skos:definition>
 	</owl:ObjectProperty>
+	
+	<owl:Class rdf:about="&fibo-sec-sec-lst;Listing">
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;isProvidedBy"/>
```

Comment:
a good argument for linking from the Exchange direction not the Listing direction